api/v1alpha1/mysqlcluster_types.go

```diff
@@ -47,6 +47,11 @@ type MySQLClusterSpec struct {
 	// +optional
 	ReplicationSourceSecretName *string `json:"replicationSourceSecretName,omitempty"`
 
+	// LogRotationSchedule is a schedule in Cron format for MySQL log rotation
+	// +kubebuilder:default="*/5 * * * *"
```

Comment:
How do you validate the given value as a valid schedule?

Reply:
How about we validate it just like k8s does?
https://github.com/kubernetes/kubernetes/blob/b33e202397319fd85e5576b35c815e8ae2f76d7e/pkg/apis/batch/validation/validation.go#L241

k8s does it with [this library](https://github.com/robfig/cron).
In this case, we validate it with Admission Webhook.

Or we validate it with regex. In this case we can do it with `kubebuilder:validation:Pattern`.

What do you think?

Reply:
Either of them is okay.

Reply:
Alright, we go for the regex.